checks/encoding.py

```diff
@@ -104,11 +104,12 @@ def checkEncodeError(lines):
 
 
 def checkEncoding(lines):
-    hasx264 = len(search('[x264 encoder:', lines))
-    hasNVENC = (len(search('[jim-nvenc:', lines)) + len(search('[NVENC encoder:', lines)))
```

Comment:
The old name might be present from old installations that never changed encoder settings, but maybe we can get rid of the detection here and live with it.